web/src/components/troubleshoot/TroubleshootContainer.tsx

```diff
@@ -104,87 +98,29 @@ class TroubleshootContainer extends Component<Props, State> {
   };
 
   render() {
-    const { app, appName } = this.props;
+    const props = {
+      //@ts-ignore
+      app: this.props.outletContext.app,
+      //@ts-ignore
+      watch: this.props.outletContext.app,
```

Comment:
do you still need these?

Reply:
yeah some components are using watch, some are using app. It will be too tedious to go through all of them right now. I'll make a separate story to clean this up.